src/handbook/development/how-we-work/staging.md

```diff
@@ -8,27 +8,29 @@ We have a staging environment running on AWS which is a scaled down replica of
 our managed FlowFuse offering, with a separate domain. Staging URL and sign in
 details can be found in the Developer Vault in 1Password.
 
-## AWS Account
+## Deployment
 
-It uses a separate AWS account ending in ..9937
+Any change to core product repositories triggers a series of GitHub Actions that results
+in staging being updated with the latest code. This can take up to 30 minutes to complete.
 
-Ben or ZJ can provision a user account for this account.
+The deploy action can be monitored [here](https://github.com/FlowFuse/helm/actions/workflows/flowforge-container.yml).
 
-The services are running in EU-West-1.
 
-## Nodes
+## AWS Account
 
-The staging environment uses one node running on a t2.small for the management
-app and a pair of t2.small nodes for the projects cluster. t2.small is the
-smallest instance that can be used with EKS.
+It uses a separate AWS account ending in ..9937. The services are running in EU-West-1.
```

Comment:
Maybe instead of writing any numbers lets add it to 1password and mention about it here?